docroot/themes/humsci/su_humsci_theme/patterns/molecules/vertical-card/vertical-card.html.twig

```diff
@@ -17,6 +17,21 @@
         {{ description }}
       </div>
     {% endif %}
+    {% if author|render|trim or year|render|trim %}
```

Comment:
probably dont need the `|render|trim` part. theres functionality occuring elsewhere that cleans up items that have no rendering.